clipper_admin/clipper_admin/kubernetes/kubernetes_metric_utils.py

```diff
@@ -0,0 +1,63 @@
+import yaml
+import os
+from kubernetes.client import V1ConfigMap
+from contextlib import contextmanager
+from kubernetes.client.rest import ApiException
+import logging
+import json
+from ..version import __version__
+
+_cur_dir = os.path.dirname(os.path.abspath(__file__))
+prom_deployment_path = os.path.join(_cur_dir, 'prom_deployment.yml')
+prom_service_path = os.path.join(_cur_dir, 'prom_service.yml')
+prom_configmap_path = os.path.join(_cur_dir, 'prom_configmap.yml')
```

Comment:
Change these file suffixes to `.yaml` to keep them consistent with the rest of the K8s yaml files